src/main.zig

```diff
@@ -213,6 +213,25 @@ pub const std_options: std.Options = .{ // MARK: std_options
 	}.logFn,
 };
 
+pub fn panicToLog(msg: []const u8, first_trace_address: ?usize) noreturn {
+	const addr = first_trace_address orelse @returnAddress();
+	std.log.err("This is a bug, please report it on the issue tracker.\n----8<---- start of panic\npanic: {s}\nerror return trace: {?f}", .{
+		msg,
+		@errorReturnTrace(),
+	});
+
+	var trace_buf: [1 << 20]u8 = undefined;
```

Comment:
This is bigger than the actual logging buffer, I'd suggest to use the same size for both.